### PR TITLE
Mesoshapes county 1.0 cleanup - Various

### PR DESCRIPTION
**Pull request by stepps00**
This PR takes care of several issues in countries that gained new features during the mesoshapes import. A rundown of what issues are taken care of, per country, below:

- New Zealand:
  - We had two records for the Outlying Areas region in this country. One of these features had extra outlying polygons, so the commit merged all geometries, brought over missing property values, and includes superseding work. More info on the outlying areas "region" of NZ can be found [here](https://en.wikipedia.org/wiki/Regions_of_New_Zealand#Areas_outside_regional_boundaries).

- Chile:
  - The original mesoshapes import included multi-part features that should have been a single Isla de Pascua record. The commit merged all polygons and includes the supersedes work for all records. 

- China:
  - The commit for China imports new features that were missed during the mesoshapes import due to poor mapping logic in the import script (similar to Thailand and Ukraine (below)).

- Pakistan/India (disputed):
  - Updated features in the Kashmir region, based on our discussion. A new geometry was created for the Kashmir record, features were deprecated, and updates to existing records in India and Pakistan were made. 

- Azerbaijan:
  - Deprecated all `admin2` features in Azerbaijan. Per conversation, all features in this country will fit into the either the `region` slot (77 features) _or_ the `macroregion` slot (10 "economic zone" features).

- Laos:
  - Two multi-part features that should have been imported as a single feature were merged. Supersedes work and deprecation was done on the smaller of the two geometries. Mapshapered and updated all necessary lat/long props.

- Serbia: 
  - Two multi-part features that should have been imported as a single feature were merged. Supersedes work and deprecation was done on the smaller of the two geometries. Mapshapered and updated all necessary lat/long props.

### CODE REVIEW

**Comment by stepps00** [comment]
Note: Commits for Ukraine features have been removed from this branch.

**Comment by stepps00** [comment]
This branch is now up-to-date and includes completed work for all countries listed above. 
All existing features in China have a single source in the `src:alt-geom`, so no additional alt-geom untangling is necessary. 

Over to @nvkelso for review.

File: data/110/869/569/3/1108695693.geojson [hunk 1]
```diff
@@ -42,7 +42,7 @@
         }
     ],
     "wof:id":1108695693,
-    "wof:lastmodified":1479508104,
+    "wof:lastmodified":1482186859,
     "wof:name":"Data Not Available",
```
Comment (nvkelso) [inline data/110/869/569/3/1108695693.geojson]: Let's call this `Siachen Glacier` to match with the disputed area that is here.

**Comment by nvkelso** [comment]
Besides the Kashmir name comment, LGTM.

**Comment by nvkelso** [comment]
Ready for PIP work.

**Comment by thisisaaronland** [comment]
Ensured `wof:parent_id/repo`. Processing in dry-run mode.

**Comment by thisisaaronland** [comment]
Dry-run successful. Processing...

**Pull request merged**